NEWS.md

```diff
@@ -1,46 +1,26 @@
-# scda.2022 0.1.3.9017
+# scda.2022 0.1.4
 
 ### Enhancements
 
 * Initialized `rcd_2022_10_13` data for 2022_10_13 release.
-* Updated `rcd_2022_10_13` with new versions of `adpc` and `adab` to make visits match, update formulas.
-* Updated `rcd_2022_10_13` with new dataset `adqlqc`.
+* Updated `rcd_2022_10_13` with new formulas.
 
 # scda.2022 0.1.3
```

Comment:
@ayogasekaram I don't think you need to do anything with the NEWS for past releases, I would just update the latest section (0.1.4) and leave <0.1.4 as is (unless things from 0.1.4 were put in the past sections by accident).